src/Render/Markdown/DocumentContext.cs

```diff
@@ -1,11 +1,14 @@
-﻿namespace D2L.Dev.Docs.Render.Markdown {
+﻿using D2L.Dev.Docs.Render.VFS;
+
+namespace D2L.Dev.Docs.Render.Markdown {
 	// TODO: Add more properties for context
 	// Some possibilities could be github repo url, list of docs repos
 	internal sealed class DocumentContext {
 
 		public string InputDirectory { get; }
 		public string OutputDirectory { get; }
 		public string DocRootRepoName { get; }
+		public SubModule SubModule { get; }

```

Comment:
Having the relevant submodule for a given file makes mapping a path for any given file easy.